### PR TITLE
Add tidb user google group link

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
add google group for tidb

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
@queenypingcap PTAL

**Comment by hanfei1991** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -55,7 +55,16 @@ for details on submitting patches and the contribution workflow.
 
 ## Follow us
 
-Twitter: [@PingCAP](https://twitter.com/PingCAP)
+### Twitter 
+[@PingCAP](https://twitter.com/PingCAP)
+
+### Mailing lists
+
+Users of TiDB trade notes on:
```
Comment (coocood) [inline README.md]: `trade notes` doesn't seem like a valid phrase.

Comment (qiuyesuifeng) [inline README.md]: it is referred from k8s.

Comment (queenypingcap) [inline README.md]: I don't think we need this sentence... After all, we only have one Google Group...

**Comment by shenli** [comment]
LGTM

**Pull request merged**